editor/src/clj/editor/code/script_compilation.clj

```diff
@@ -77,10 +77,14 @@
   (when-let [workspace->extensions (resource-kind->workspace->extensions resource-kind)]
     (workspace->extensions workspace)))
 
-(defn script-property-edit-type [workspace prop-type resource-kind]
-  (if (= resource/Resource prop-type)
-    {:type prop-type :ext (resource-kind-extensions workspace resource-kind)}
-    {:type prop-type}))
+(defn script-property-edit-type
+  ([workspace prop-type resource-kind & [script-property-type]]
+   (let [base-map (if (= resource/Resource prop-type)
+                    {:type prop-type :ext (resource-kind-extensions workspace resource-kind)}
+                    {:type prop-type})]
+     (if script-property-type
+       (assoc base-map :script-property-type script-property-type)
```

Comment:
I tried approach with new `g/MsgUrl` type, but then I can't add icons for the other fields because in such case I'm adding them all over the editor, not only the go propertiies. So, I decided to go with `edit-type` extra field